save-cloud-common/src/jvmMain/kotlin/com/saveourtool/save/entities/Contest.kt

```diff
@@ -23,6 +24,7 @@ data class Contest(
     @Contextual
```

Comment:
BTW, I don't think we need these annotations here

Reply:
Yes, we used to need it when `Contest` was in `commonMain`. Will fix it